docs/meetings/2023/TechArea20230718.md

```diff
@@ -38,21 +38,28 @@ Release
 -   AI (Mat): Release XRootD 5.6+
 -   AI (BrianL): Release xrootd-multiuser-2.1.3+
 -   AI (Matt): osg-ca-scripts missing dependency on perl-filetest.
-    We need this done before we can build EL9 tarballs
--   AI (Matt): Release oidc-agent-4.4.2+
+    We need this done before we can build EL9 tarballs - no longer included with base perl packages in EL9
+-   AI (Matt): Release oidc-agent-4.4.2+.
+    SRPMs no longer provided by upstream, github is not properly organized to use as source.
+    Either a spec file in the github, or return to releasing the srpm
 
-OSG 3.7 tasks
+OSG 23 tasks
 
 -  What's the next step in the Yubikey setup for promoting packages?
+   Per-developer key, packages get re-signed when promoted from minefield to testing
+   Need to:
+     - Update script OSG Promote
+     - Generate promotion key
+     - Create Yubikey for each developer (4 or 5)
+     - Workflow update: Will need to sign packages locally, since you can't use a Yubikey on Moria
 -  AI (Matt): Build buildsys-macro packages
 -  AI (BrianL): build package list
 -  AI (Matt): update release scripts to accommodate OSG 23
+   New args: <date> <series 1> ... <series n> (as long as it sorts)
```

Comment:
```suggestion
   New args: <YYMMDD> <series 1> ... <series n> (as long as it sorts)
```

May as well keep the same date format. I think it makes sense for the dated OSG 23 tags to look like the following: `osg-23-el9-release-230503`